policyengine_canada/variables/gov/cra/tax/income/income_tax_after_refundable_credits.py

```diff
@@ -0,0 +1,13 @@
+from policyengine_canada.model_api import *
+
+
+class income_tax_after_refundable_credits(Variable):
```

Comment:
For consistency with US+UK (also filename)
```suggestion
class income_tax(Variable):
```